BeeSwift/AppDelegate.swift

```diff
@@ -153,22 +153,16 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
     func application(_ application: UIApplication, continue userActivity: NSUserActivity, restorationHandler: @escaping ([UIUserActivityRestoring]?) -> Void) -> Bool {
         if #available(iOS 12.0, *) {
```

Comment:
While here, how about turning this line into a `guard`?

Reply:
I don't think this behaves quite like a guard, because we still want to inspect `userActivity.userInfo?` later even if it's not satisfied?

Reply:
Of course the function will need to return something when running on lower than ios 12. The guard's else provides for this.

The meat of the function though happens inside that if and using a `guard` here should allow one to compose much more legible code.

See https://github.com/raywenderlich/swift-style-guide#golden-path